test/tests/functional/pbs_node_buckets.py

```diff
@@ -444,7 +444,7 @@ def test_not_buckets(self):
         j_c1 = Job(TEST_USER, attrs)
         jid_c1 = self.server.submit(j_c1)
         self.server.expect(JOB, {'job_state': 'R'}, id=jid_c1)
-
+        self.scheduler.log_match(jid_c1 + ';Chunk: ' + '1430:ncpus=1:color=orange', n=10000)
```

Comment:
No need to plus two static strings together.  Just merge them into one string.

Reply:
done. Also added latest test run logs